mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -150,6 +151,19 @@ public void testScanTable() throws IOException {
     Assert.assertArrayEquals(new Object[] {"Alice", 0L}, descRows.get(2));
   }
 
+  @Test
+  public void testDescribeTable() throws IOException {
```

Comment:
It would be good to have a test for:
```
CREATE TABLE alma (korte INT COMMENT 'barack')
```

I think the change need to be done here:
https://github.com/apache/iceberg/blob/969a1ffc8d7392cc4c54eba156647c448757b45e/mr/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergSerDe.java#L135-L153

Reply:
I have a few tests in TestHiveIcebergStorageHandlerLocalScan.

Reply:
Do we create these tables with comments for the colums?

Reply:
Yes, I added some dummy comments

Reply:
Opps... missed that :( Sorry

Reply:
Have we checked that the Hive table contains the comments?

Reply:
Yes, the comments are now available for each column.